main/main.go

```diff
@@ -174,7 +177,20 @@ func main() {
 
 	relayerHealth := createHealthTrackers(&cfg)
 
-	messageHandlerFactories, err := createMessageHandlerFactories(logger, &cfg)
+	deciderClient, err := createDeciderClient(cfg.DeciderURL)
```

Comment:
This function is actually creating a `grpcClient`

Reply:
I actually used to call it a `grpcClient`, but Cam had me change it to `deciderClient`. I think Cam is right because this client instance is connected to a specific URL, and that URL will almost certainly be serving only the decider API (at least, as far as this context/scope needs to be aware of). I think the only situation in which it would make sense to generalize to `grpcClient` would be in the context of a client instance that will interact with a server using multiple different gRPC `service` interfaces.

Reply:
So my issue here is that the struct `DeciderServiceClient` is created from this object in `NewMessageHandlerFactory`. We could call this one `deciderConnection`, but we should really distinguish them.

Reply:
Thank you, this is much clearer now :)